lib/ansible/modules/packaging/os/snap.py

```diff
@@ -141,33 +142,42 @@ def remove_snap(module, snap_name):
     rc, out, err = module.run_command(cmd, check_rc=False)
 
     if rc == 0:
-        # snap has been removed
-        module.exit_json(msg="The snap '%s' has been removed" % str(snap_name), changed=True)
+        # Snap has been removed
+        return True
     else:
         # Something went wrong
         module.fail_json(msg=err)
 
 
 def main():
     module_args = dict(
-        name=dict(type='str', required=True),
+        name=dict(type='list', required=True),
         state=dict(type='str', required=False, default='present', choices=['absent', 'present']),
         classic=dict(type='bool', required=False, default=False)
     )
     module = AnsibleModule(argument_spec=module_args)
-    snap_name = module.params['name']
+    snap_names = module.params['name']
     state = module.params['state']
 
-    # Test if snap exists
+    # Check if snaps are valid
+    for snap_name in snap_names:
     rc, out, err = snap_exists(module, snap_name)
```

Comment:
broken indent

Reply:
Fixed in https://github.com/ansible/ansible/commit/273f5476d89538f8b814f1379eeef57aceb2b0b9